lib/request/post.js

```diff
@@ -57,7 +57,7 @@ const actionPost = (action, data, config) => authData => {
   if (summary != null) data.summary = data.summary || summary
   if (baserevid != null) data.baserevid = data.baserevid || baserevid
   if (tags != null) data.tags = data.tags || tags.join('|')
-  data.maxlag = maxlag || 5
+  if (maxlag != null) data.maxlag = maxlag || 5
```

Comment:
a strict inequality check (`!==`) is required so that an undefined value still gets the default value
```suggestion
  if (maxlag !== null) data.maxlag = maxlag || 5
```

Reply:
You're right! Thanks